src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -292,11 +292,6 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None, input_meta=None):
                 meta.max_memory = 0.5
             if not hasattr(meta, 'nfiles'):
                 meta.nfiles = 1
-            if meta.nfiles == 1 and meta.nfiles > 1:
-                log.writelog('WARNING: Strange behaviors can occur if you set '
-                             'nfiles to 1. If your computer has enough RAM to '
-                             'load many/all of your Stage 2 files, it is '
-                             'strongly recommended to increase nfiles.')
```

Comment:
What's the mentality for removing this? You figure all the issues with only using one file were solved by the recent option to avoid independent batches? I'd assume performance would still be significantly better with more than one file since that allows more averaging. Or is the issue that HST round-trip scanning wasn't working with larger batches or something?

Reply:
Yes, it seemed like the warning is no longer necessary now that you've implemented the recent fix.  Maybe the better solution would be to change the warning to encourage users to set `indep_batches = False` in the S3 meta (after checking to see if it is set to True).